src/index.js

```diff
@@ -34,7 +34,7 @@ class DbConnector {
     }
 
     // find & connect to mongo DBs
-    var mongoConfigs = configs.filter((x) => {return x.connectionString.startsWith('mongodb://')})
+    var mongoConfigs = configs.filter((x) => {return /^mongodb(\+srv)?:\/\/.+/.test(x.connectionString)})
```

Comment:
I don't think it's necessary to use a regex, let's keep it simple.

```suggestion
    var mongoConfigs = configs.filter((x) => {return x.connectionString.startsWith('mongodb')})
```